plugins/modules/scc_product.py

```diff
@@ -72,6 +71,22 @@ def main():
     module.task_timeout = 4 * 60
 
     with module.api_connection():
+        try:
+            statuses = module.foremanapi.resource('ping').call('statuses')
+            plugins = statuses['results']['foreman']['plugins']
+            scc_version = '1.0.0'  # fallback
+            for plugin in plugins:
+                if 'foreman_scc_manager' in plugin:
+                    scc_version = plugin.split(',')[1]
+        except Exception:
+            scc_version = '1.8.6'
```

Comment:
I think I got the logic here a tad wrong: the `try/except` is to support Foreman versions that didn't have the `statuses` API yet, but then we should not fall-back to using 1.8.6 as the version.

```suggestion
        scc_version = '1.0.0'  # fallback
        try:
            statuses = module.foremanapi.resource('ping').call('statuses')
            plugins = statuses['results']['foreman']['plugins']
            for plugin in plugins:
                if 'foreman_scc_manager' in plugin:
                    scc_version = plugin.split(',')[1]
        except Exception:
            pass
```

That should be cleaner